source/_integrations/tplink.markdown

```diff
@@ -35,50 +38,87 @@ There is currently support for the following device types within Home Assistant:
+| LB110            | ✅                        | ✅ <!-- whole series should to work -->    |                                          |
+| LB120            | ✅                        | ✅ <!-- whole series should to work -->    |                                          |
+| LB130            | ✅                        | ✅ <!-- whole series should to work -->    |                                          |
+| LB230            | ✅                        | ✅ <!-- whole series should to work -->    |                                          |
 
 ### Light Strips
 
-- KL400
-- KL420
-- KL430
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
+|------------------|---------------------------|--------------------------------------------|------------------------------------------|
+| KL400            | ✅                        | ❓ <!-- not mentioned as of 2022-05-01 --> |                                          |
+| KL420            | ✅                        | ❓ <!-- not mentioned as of 2022-05-01 --> |                                          |
+| KL430            | ✅                        | ❌                                         |                                          |
+| KL430E           | ❓                        | ❌                                         |                                          |
+
+### Range Extender
+
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
+|------------------|---------------------------|--------------------------------------------|------------------------------------------|
+| RE270            | ❓                        | ✅                                         |                                          |
+| RE370            | ❓                        | ✅                                         |                                          |
+| RE350K           | ❓                        | ✅                                         |                                          |
+
+### Cameras
+
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
+|------------------|---------------------------|--------------------------------------------|------------------------------------------|
+| All Kasa cameras | ❓                        | ❌ <!-- as of 2022-05-01               --> |                                          |
+
+### Smart Routers
+
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
+|------------------|---------------------------|--------------------------------------------|------------------------------------------|
+| SR20             | ❓                        | ❌                                         |                                          |
```

Comment:
None of these are supported.  We should say all Range Extenders, Cameras, and Smart Routers are not supported instead of listing them out.